_data/vendors.yml

```diff
@@ -126,6 +126,14 @@
   updated_at: 2018-10-17
 
 - name: Figma
+  url: https://www.frontapp.com
+  base_pricing: $19 per u/m
+  sso_pricing: $99 per u/m
+  percent_increase: 421%
+  pricing_source: https://www.frontapp.com/pricing
+  updated_at: 2021-05-12
+  
+  - name: Front
```

Comment:
You've added this in the wrong place by the looks of it.

Reply:
Fixing in my patch